server/game/game.js

```diff
@@ -1232,13 +1232,6 @@ class Game extends EventEmitter {
             name: this.name,
             owner: _.omit(this.owner, ['blocklist', 'email', 'emailHash', 'promptedActionWindows', 'settings']),
             players: playerSummaries,
-            rings: {
-                air: this.rings.air.getState(activePlayer),
-                earth: this.rings.earth.getState(activePlayer),
-                fire: this.rings.fire.getState(activePlayer),
-                void: this.rings.void.getState(activePlayer),
-                water: this.rings.water.getState(activePlayer)
-            },
```

Comment:
What was this change for?

Reply:
The build is failing on lint because the activePlayer is no longer a used variable.

Reply:
Ah sorry, just saw in your description why it was removed.  It still needs fixing so that `activePlayer` is removed as it is no longer a used variable.